docs/distributions/fedora/installation.md

```diff
@@ -1,6 +1,8 @@
 # Download the latest safe release
 
-Many thanks to Mike for building. You can download a live iso [here](https://github.com/mikeeq/mbp-fedora)
+Many thanks to Mike for building. You can download a live iso [here](https://github.com/mikeeq/mbp-fedora).
+
+If you need a more updated kernel, audio working out of the box, LUKS, or bluetooth support use the iso fomr [here](https://github.com/sharpenedblade/t2linux-fedora-iso/releases). Follow the instalation instructions below until step 5, then install like a normal Fedora iso. If you do not know how to install Fedora normally, read [this](https://docs.fedoraproject.org/en-US/fedora/latest/install-guide/). Remember to follow the [wifi guide](https://wiki.t2linux.org/guides/wifi/).
```

Comment:
Spelling of from is written as fomr here. Please correct that.

Reply:
Also, B capital in Bluetooth, and wifi to be written as Wi-Fi

Reply:
The link now is https://wiki.t2linux.org/guides/wifi-bluetooth/

Reply:
I forgot about this PR, the only benefit of my kernel is faster updates now, so I removed most of that section.